cmd/serve.go

```diff
@@ -13,6 +15,20 @@ var serveCmd = &cobra.Command{
 It will accept HTTP requests on localhost:8080 by default.
 Use the --port and --host flags to change the listening option.`,
 	RunE: func(cmd *cobra.Command, args []string) error {
+		// The serve method starts all the jobs systems associated with the created
+		// instances.
+		//
+		// TODO: on distributed stacks, we should not have to iterate over all
+		// instances on each startup
+		ins, err := instance.List()
+		if err != nil && !couchdb.IsNoDatabaseError(err) {
+			return err
+		}
+		for _, in := range ins {
+			if err := in.StartJobSystem(); err != nil {
+				return err
+			}
+		}
```

Comment:
You sure about that. I put it in `serve.go` knowing that it is temporary.

The thing is that we rely on one job system per instance since they are namespaced by domains (broker and queues...).